content/accessibility/tooltip-alternatives.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Tooltip alternatives
+---
+
+## Overview
+
+Tooltips are often used to convey information. However, tooltips are rarely appropriate, and its misuse can result in a myriad of accessibility issues.
+Always consider **not** using a tooltip for an improved user experience.
+
+## Why?
+
+- Tooltips are hidden by default making it easy to miss, so it should never be used to convey critical information.
+- Tooltips are never accessible on mobile devices.
+- Tooltips should **never** be set on static elements. Tooltips on static elements are not accessible to keyboard users and screen reader users.
+
+## Guidelines 
+
+### For designers
+
+Only include tooltips on your design as a last resort. Never include tooltips on static elements. 
+
+If you have an appropriate tooltip in your design, be sure to annotate the tooltip carefully with the preferred type (for example, label or description) to ensure proper semantics.
+
+### For developers
+
+If you come across a design or a page with a tooltip on a static element, remove it. Consult a designer for an alternative way of conveying the information.
+
+If you determine that tooltips are appropriate for your usecase, make sure to carefully follow guidelines set in [PVC: Tooltip component](https://primer.style/view-components/components/alpha/tooltip).
+
+## What are alternatives to tooltips?
+
+### 1. Always persist the content
+
+If possible, persist the content so it's available rather than using a tooltip which hides content by default. This ensures that the content is always discovered and surfaced to consumers regardless of device.
```

Comment:
One example that might be relevant is the "Pin Issue" button in an issue sidebar describing the max number of issues that could be pinned.

It could be reworked to say "Pin issue (max 3)" or something.